lib/search/query.rb

```diff
@@ -85,11 +85,11 @@ def fetch_spell_checks(search_params)
 
     def process_elasticsearch_errors
       yield
-    rescue Elasticsearch::Transport::Transport::Errors::BadRequest => e
+    rescue Elasticsearch::Transport::Transport::Errors::InternalServerError => e
```

Comment:
What's the API change behind this? It feels odd for it not to return BadRequest here...

Reply:
This means that if either of the below two errors occurs ES is now returning a `500` response, which ES 1.7 returned a `400` response. 

I don't have any further details on why this change was made. I can investigate further if you think it is important?